crates/worker/src/partition/leadership.rs

```diff
@@ -917,7 +945,7 @@ impl Future for SelfAppendFuture {
     type Output = ();
 
     fn poll(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output> {
-        let append_result = ready!(self.0.poll_unpin(cx));
+        let append_result = ready!(self.commit_token.poll_unpin(cx));
```

Comment:
Thanks ❤️

Reply:
Soon, I'll start a petition to forbid tuple structs 😅